src/System.Device.Gpio/System/Device/Gpio/Drivers/UnixDriver.cs

```diff
@@ -14,6 +21,10 @@ protected UnixDriver()
             }
         }
 
+        /// <summary>
+        /// Static factory method
+        /// </summary>
+        /// <returns>An instance of GpioDriver, depending on which one fits</returns>
```

Comment:
Now that I think of it, given that we have a factory pattern we missed adding a platform check to it to throw when called from Windows. You don't have to fix it in this PR if you don't want to but just wanted to mention it

Reply:
If you try this on Windows, both LibGpiodDriver and SysFsDriver will fail, so you already end up with an exception.

Reply:
you are correct, I guess it is just really a matter of making the code more readable as this way it would be straight obvious just by looking at the UnixDriver.cs that this driver is not supported on WIndows (although I guess you could infer it by looking at the file name 😆 )